zulia-client/src/main/java/io/zulia/client/command/builder/StandardQuery.java

```diff
@@ -5,7 +5,7 @@
 
 import java.util.List;
 
-public abstract class StandardQuery implements QueryBuilder {
+public abstract class StandardQuery<T extends StandardQuery> implements QueryBuilder {
```

Comment:
Nice generic action.

Reply:
Took me awhile to figure it out but now you can chain things like
```
FilterQuery fq2 = new FilterQuery("a b c").setDefaultOperator(ZuliaQuery.Query.Operator.AND).exclude().addQueryFields("title", "abstract");
```

Before some methods returned StandardQuery so chaining would result in StandardQuery and inability to use FilterQuery/ScoredQuery methods or chained assignment